gdk-sys/src/lib.rs

```diff
@@ -14,6 +14,7 @@ use libc::{c_int, c_char, c_double, c_void, c_uint, c_uchar, c_ulong, c_float};
 pub use cairo_ffi::cairo_rectangle_int_t as GdkRectangle;
 
 pub use glib_ffi::{gboolean, gpointer, gsize, GType};
+pub use cairo_ffi::{cairo_t, cairo_content_t, cairo_pattern_t, cairo_region_t, cairo_surface_t};
```

Comment:
I'm not sure reexporting a lot of types is needed. Even exporting `GdkRectangle` above might not have been a good idea.